pkg/agent/nodeportlocal/rules/types.go

```diff
@@ -16,6 +16,8 @@ package rules
 
 // PodNodePort contains the Node Port, Pod IP, Pod Port and Protocols for NodePortLocal.
```

Comment:
```suggestion
// PodNodePort contains the Pod namespaced name, Node Port, Pod IP, Pod Port and Protocol for NodePortLocal.
```
or remove the comment? I feel it's self-explanation and no need to add comment for `PodNodePort`.

Reply:
Updated, I didn't remove it as a struct's comment may be useful when generating doc or when IDE references the struct.